src/Sulu/Bundle/ContactBundle/Behat/ContactContext.php

```diff
@@ -22,12 +22,16 @@
  */
 class ContactContext extends BaseContext implements SnippetAcceptingContext
 {
+    private static $emailTypeId = 100;
```

Comment:
The don't. They start at `1`. This can bee seen in the DataFixtures of the ContactBundle. The line you commented is for the behat tests and considered when "Given an email type ... exists" occurs in a behat scenario. Maybe we could switch to a `SequencedGenerator` at this point. Is that possible with doctrine?